src/main/java/app/coronawarn/verification/config/PostSizeLimitFilter.java

```diff
@@ -0,0 +1,35 @@
+package app.coronawarn.verification.config;
+
+import java.io.IOException;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+@Component
+public class PostSizeLimitFilter extends OncePerRequestFilter {
+
+  @Value("${server.max-post-size:10000}")
+  private long maxPostSize;
```

Comment:
I think we should use the VerificationApplicationConfig Class for the config property.
The rest looks good to me.

@martinmuellerrohde if you allow me to push in your fork I can add these changes. Otherwise we would have to create a new PR.

Reply:
hey Felix, sorry for the delay - I've invited you as a colloborator in my fork now.
Feel free to use VerificationApplicationConfig now.

Reply:
Hi @martinmuellerrohde ,
thanks for your invitation. I've made the changes as I have proposed.